design/comments.go

```diff
@@ -60,16 +60,10 @@ var createCommentAttributes = a.Type("CreateCommentAttributes", func() {
 })
 
 var commentRelationships = a.Type("CommentRelations", func() {
-	a.Attribute("created-by", commentCreatedBy, "This defines the created by relation")
+	a.Attribute("creator", relationGeneric, "This defines the creator of the comment")
```

Comment:
I can understand why you want to change the db as well but I wanted you to realize that it wasn't necessary.

This is required from the issue:

```json
"creator": {
  "data": {
    "id": "746de3a4-d8c4-4231-8b9d-3425d8d3c374",
    "type": "users",
    "links": {
      "self": "http://...."
    }
  }
}
```

Although the `relationGeneric` can sure represent this structure it will make the swagger documentation less readable because you will not know what to expect. The `identityRelationData` on the other hand had a fixed `type` field which will help you understand what will be included in that `creator` field.

I would suggest to 

1. rewrite the `commentCreatedBy` to `commentCreator`
2. swap the `identities` type out for `users`
3. add an `a.Attribute("links", genericLinks)`

WDYT?